docs/modelserving/disable_toplevel_virutalhost/README.md

```diff
@@ -0,0 +1,84 @@
+# Deploy model without istio top level virtual service
+
+To disable top level virtual service add the flag `"disableIstioVirtualHost": true` under **ingress** config in inferenceservice configmap.
```

Comment:
Can you add maybe a sentence or two explaining why this is important/when you want to do this/what the implications are?

Reply:
I think that would help to tell a story for these docs for someone wanting to learn about this